README.md

```diff
@@ -1,3 +1,22 @@
-# Guidelines
+#GUIDELINES WIKI
+**Aspiring to be a World-Class software developer/engineer?** This is a guide where you will get engaged on the great values,
```

Comment:
What do you think about splitting this up into two sentences to make it a bit snappier?

e.g.
```
These guidelines will introduce you to the values, best software development practices...the Andela community has to offer. 
With them, you'll be well equipped in your journey....
```